src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -30,8 +33,10 @@
 public class LoginUtils {
     private static final Pattern LOGIN_PATTERN = Pattern.compile("\"bgcdw_login_form\" action=\"(.*)\"");
     private static final Pattern DATA_PATTERN = Pattern.compile("\"src\": \"([^\"]*)\".*}, \\{(.*)}");
-
     private static final Map<String, String> FORCED_PARAMS = new HashMap<>();
+    static CookieManager cookieManager;
+    static boolean captcha;
```

Comment:
you need to scope your changes better, just having these 2 random variables here floating is going to be a future pain in maintainability, keep the cookie manager scoped to where it was